poetry/console/commands/init.py

```diff
@@ -67,9 +68,23 @@ def handle(self):
         from poetry.utils._compat import Path
         from poetry.utils.env import SystemEnv
 
+        original_toml = None
+
         if (Path.cwd() / "pyproject.toml").exists():
-            self.line("<error>A pyproject.toml file already exists.</error>")
-            return 1
+            with (Path.cwd() / "pyproject.toml").open() as toml_file:
```

Comment:
We can now move this to use `PyProjectTOML` now that core has been updated. We can also deal with that in a later change if required. Let me know what is preferred.

https://github.com/python-poetry/poetry-core/blob/a056c464ebaf0f1638b6b0be97e4be20dc4a678d/poetry/core/pyproject/toml.py#L13